app/code/Transoft/Catalog/registration.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Why do we need second module? Please move all data to **Transoft_Blog**

Reply:
fixed